src/generate/generic.rs

```diff
@@ -32,6 +32,21 @@ pub struct Reg<U, REG> {
 
 unsafe impl<U: Send, REG> Send for Reg<U, REG> {}
 
+impl<U, REG> Reg<U, REG>
+where
+    U: Copy,
+{
+    /// Gets a memory address of a register.
```

Comment:
Wording, maybe something like

    Returns the underlying memory address of register

Reply:
Adjusted.